### DISCUSSION TITLE
LSP error message is cut off; how do I read it?

### DISCUSSION THREAD

**Discussion by thecristidima** [LunarVim/LunarVim#4535] [Q&A] [answered]
I'm setting up LunarVim on my machine and I want to use it for C# development.

I managed to use it without any issues for smaller projects, but when I open a larger one I get the following message at the bottom

`LSP[csharp_ls][Log] csharp-ls: msbuildWorkspace.Diagnostics: [Failure] Msbuild failed when processing t...'`

I assume this is an issue on my part, but I'm not sure how to get the whole log or if this error is written to some log file (ideally).

**Comment by frolvanya** [comment, marked as answer]
You should be able to find an issue here: `~/.local/state/lvim/lsp.log`
